tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -194,12 +194,7 @@ public function testBucketEvents(string $devBranchType, string $userRole): void
         $this->assertEventWithRetries($this->_testClient, $assertCallback, $query);
 
         $bucketEvents = $this->_testClient->listBucketEvents($bucketId);
-        if ($devBranchType === ClientProvider::DEFAULT_BRANCH) {
-            // there are already events in production
-            $this->assertGreaterThan(2, $bucketEvents);
-        } else {
-            $this->assertCount(2, $bucketEvents);
-        }
+        $this->assertGreaterThan(2, $bucketEvents);
```

Comment:
tim ze sa pouziva stale jedna branch a nevytvara nova tak je to ako v maine a tych eventov tam moze byt viac